lib/src/Base/Graph/Contour.cxx

```diff
@@ -35,20 +36,73 @@ static const Factory<Contour> Factory_Contour;
 /* Default constructor */
 Contour::Contour()
   : DrawableImplementation()
-  , drawLabels_(true)
+  , drawLabels_(ResourceMap::GetAsBool("Contour-DrawLabels"))
+  , isFilled_(ResourceMap::GetAsBool("Contour-IsFilled"))
+  , colorBarPosition_(ResourceMap::GetAsString("Contour-ColorBarPosition"))
+  , isVminUsed_(false)
+  , vmin_(0)
+  , isVmaxUsed_(false)
+  , vmax_(0)
+  , colorMap_(ResourceMap::GetAsString("Contour-ColorMap"))
+  , alpha_(ResourceMap::GetAsScalar("Contour-Alpha"))
+  , norm_(ResourceMap::GetAsString("Contour-Norm"))
+  , extend_(ResourceMap::GetAsString("Contour-Extend"))
 {}
 
-/* Default constructor */
-Contour::Contour(const UnsignedInteger dimX,
-                 const UnsignedInteger dimY,
+/* Constructor with parameters */
+Contour::Contour(const Sample & x,
+                 const Sample & y,
                  const Sample & data,
+                 const Point & levels,
+                 const Description & labels,
+                 const Bool drawLabels,
                  const String & legend)
   : DrawableImplementation(data, legend)
+  , x_(x)
+  , y_(y)
+  , levels_(levels)
+  , labels_(labels)
+  , drawLabels_(drawLabels)
+  , isFilled_(ResourceMap::GetAsBool("Contour-IsFilled"))
+  , colorBarPosition_(ResourceMap::GetAsString("Contour-ColorBarPosition"))
+  , isVminUsed_(false)
+  , vmin_(0)
+  , isVmaxUsed_(false)
+  , vmax_(0)
+  , colorMap_(ResourceMap::GetAsString("Contour-ColorMap"))
```

Comment:
when RM entries are used to initialize an attribute (that can be set afterwards), the Default prefix is added, eg Contour-DefaultColorMap

Reply:
Done